features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/HandledInternalNotifyScenario.swift

```diff
@@ -19,7 +19,6 @@ import Bugsnag
                 ["method":"bar()", "file":"parser.js"],
                 ["method":"yes()"]
             ]
-            report.attachCustomStacktrace(frames, withType: "unreal")
```

Comment:
there is a maze runner expectation on this type that is failing

Reply:
Yep - left this in the draft state overnight as the mazerunner scenario was using one of the removed APIs. Planning on updating the scenario/assertions today.